testsuite/tests/unwind/Makefile

```diff
@@ -0,0 +1,19 @@
+default:
+	@if $(BYTECODE_ONLY); then : ; elif [ $(SYSTEM) = macosx ]; then $(MAKE) native_macosx_tests; fi
+
+native_macosx_tests:
+	@printf " ... testing 'unwind_test':"; \
+        $(MAKE) clean ; $(MAKE) unwind_test && \
+	./unwind_test >/dev/null 2>&1 && echo " => passed" || echo " => failed"
+
+unwind_test:
+	@ocamlopt -c -opaque mylib.mli
```

Comment:
@mshinwell is right, these should be `$(OCAMLOPT)` because you can't assume that OCaml is installed.